docs/source/terms_and_model.rst

```diff
@@ -700,3 +705,145 @@ subclasses, but are still treated as variation.
 
 .. _GitHub issue: https://github.com/ga4gh/vr-spec/issues
 .. _genetic variation: https://en.wikipedia.org/wiki/Genetic_variation
+
+
+VariationSet
+############
+
+**Biological definition**
+
+Sets of variation are used widely, such as sets of variants in dbSNP
+or ClinVar that might be related by function. 
+
+**Computational definition**
+
+An unconstrained set of Variation objects or references.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Identifier of the VariationSet.
+   * - type
+     - string
+     - 1..1
+     - Variation type; MUST be "VariationSet" (default)
+   * - members
+     - :ref:`Variation`\[] or :ref:`CURIE`\[]
+     - 0..*
+     - List of Variation objects or identifiers. Attribute is
+       required, but MAY be empty.
+
+
+**Implementation Guidance**
+
+* The VariationSet identifier MAY be computed as described in
+  :ref:`computed-identifiers`, in which case the identifier
+  effectively refers to a static set because a different set of
+  members would generate a different identifier.
+* `members` may be specified as Variation objects or CURIE
+  identifiers.
+* CURIEs MAY refer to entities outside the `ga4gh` namespace.
+  However, objects that use non-`ga4gh` identifiers MAY NOT use the
+  :ref:`computed-identifiers` mechanism.
+* VariationSet identifiers computed using the GA4GH
+  :ref:`computed-identifiers` process do *not* depend on whether the
+  Variation objects are inlined or referenced, and do *not* depened on
+  the order of members.
+* Do not use VariationSet for Haplotypes or Genotypes. VRS will soon
+  (2020 Q2?) support dedicated classes that will provide appropriate
+  constraints for these data.
```

Comment:
The current PR already assumes addition of Haplotype and Genotype in `1.1` (see my above comment on the `future_plans.rst` changes). We should omit this to match.
```suggestion
```